### PR TITLE
Idiomaspuentes main ip

### PR DESCRIPTION
**Pull request by foxprogs**
## Changes proposed in this pull request:
 -
 -
 -

`Has the changelog been added to what is being done in this pull request`

### CODE REVIEW

**Review by Pavel064 — APPROVED** [review]

File: src/config/locales/es-419/translation.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
+{
```
Comment (Valyukhov) [inline src/config/locales/es-419/translation.json:1]: muy bien!

**Review by Valyukhov — APPROVED** [review]
looks good and it's work

File: src/config/locales/es-419/translation.json (hunk 1 above)
Comment (foxprogs) [reply src/config/locales/es-419/translation.json:1]: =))

File: src/components/SupportTA/SupportTA.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useContext, useMemo } from 'react';
+
+import { Card, CardContent, useContent, useCardState } from 'translation-helps-rcl';
+
+import { AppContext } from '../../context';
+
+export default function SupportTN({
```
Comment (abelpz) [inline src/components/SupportTA/SupportTA.js:7]: The name of the function should be SupportTA. My bad.

**Pull request merged**